app/models/hvd/Hvd.scala

```diff
@@ -82,12 +82,18 @@ case class Hvd (cashPayment: Option[CashPayment] = None,
   def isComplete: Boolean = {
     Logger.debug(s"[Hvd][isComplete] $this")
     this match {
-        case Hvd(Some(_), Some(pr), _, Some(_), Some(_), Some(true), Some(_), Some(_), _, _, true)
-          if pr.items.forall(item => item != Alcohol && item != Tobacco) => true
-        case Hvd(Some(_), Some(_), Some(_), Some(_), Some(_), Some(true), Some(_), Some(_), _, _, true) => true
-        case Hvd(Some(_), Some(pr), _, Some(_), Some(_), Some(false), _, Some(_), _, _, true)
-          if pr.items.forall(item => item != Alcohol && item != Tobacco) => true
-        case Hvd(Some(_), Some(_), Some(_), Some(_), Some(_), Some(false), _, Some(_), _, _, true) => true
+        case Hvd(Some(cp), Some(pr), _, Some(_), Some(_), Some(true), Some(_), Some(_), _, _, true)
+          if pr.items.forall(item => item != Alcohol && item != Tobacco) & cp.isCashPaymentsComplete => true
```

Comment:
Did you intend to use a single & here? Would usually use double
https://stackoverflow.com/questions/8691693/bitwise-operator-and-single-ampersand

Reply:
Thanks Owen, I have amended and pushed that now.

Reply:
Think you missed one :)